landoapi/api/transplants.py

```diff
@@ -171,7 +171,9 @@ def _assess_transplant_request(phab, landing_path):
     reviewers = {
         revision["phid"]: get_collated_reviewers(revision) for revision, _ in to_land
     }
+    is_secure = revision_is_secure(revision, phab)
 
+    # fmt: off
```

Comment:
Why? This code seems pretty basic, I don't see why it warrants special formatting?

Reply:
This change was reverted with the old warning behaviour.